specification/compiler/index.ts

```diff
@@ -18,19 +18,21 @@
  */
 
 import Compiler from './compiler'
-// import validateRestSpec from './steps/validate-rest-spec'
+import validateRestSpec from './steps/validate-rest-spec'
 import addInfo from './steps/add-info'
 import addDescription from './steps/add-description'
+import validateModel from './steps/validate-model'
+import removeIncompleteEndpoints from './steps/remove-incomplete-endpoints'
 
 const compiler = new Compiler()
 
 compiler
   .generateModel()
   .step(addInfo)
-  // validateRestSpec is not enabled until the RequestBase
-  // properties gets moved to a custom behavior
-  // .step(validateRestSpec)
+  .step(validateRestSpec)
+  .step(removeIncompleteEndpoints)
   .step(addDescription)
+  .step(async (model, spec) => validateModel(model, false))
```

Comment:
I'm wondering if we can keep the step declaration like the others, and choose to `failHard` based on an env variable or command line parameter.

Reply:
I initially went this way so that validation could easily be extracted in a different context.

But since we've decided that it will stay there and ultimately output validation errors in an additional file that will consume these errors, it doesn't make sense anymore. I changed it to be consistent with other functions and have a nice-looking pipeline ;-)